src/content/whats-new/2022/04/whats-new-4-20-removal-of-estimated-costs.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Simplified usage and cost tracking
+summary: We've simplified your end-of-month costs.
```

Comment:
Given that this announcement is going out before the actual change - can we change this to "We're simplifying .."